src/bw_api.py

```diff
@@ -71,7 +71,8 @@ def get_assignment_config(cid, aid):
 		if resp.status_code == 200:
 			ret_data = resp.json()
 			return ret_data["data"]
-	except: pass
+	except Exception as e:
```

Comment:
I'm guessing you looked at an old commit, the current code has something like this:
`logging.error("start_grading_run(cid={}, aid={}, netid={}): {}".format(cid, aid, netid, repr(e)))
`

Reply:
ohhh yeah